### ISSUE TITLE
Using Quantum Random Number Number Generator , to improve on the aspect of randomness

### ISSUE THREAD

**Issue by Abhigyan-Mishra** [SirZeck/pw-gen#1]
There is a scope of using QRNG , in order to tackle with the pseudo randomness .

**Comment by SirZeck** [comment]
Yes, there is. This is a mockup build for what is going soon to be a bigger project which will be generating password which is unbreakable by quantum computers in the coming years. This is a first step toward that. Stay tuned...

**Comment by Abhigyan-Mishra** [comment]
I will take this as a challenge , i will heck you .
 I will burn your billion dollar company(The one you will make by referencing your password generator as unbreakable ) to the ground.
 
 
 
 _\\/|(/_
   >_   _ /
   (_)-(_)
    ) o (
    \ = /
     |W|
     | |
   __| |__
  / \ u / \
 |   `-'   |
 |__|   |__|
  |||ADM|||
  |||   |||
  |||   |||
  |||   |||
  |||   |||
  |||   |||
 /  |___/  \
 ||(|   \)||
  \\| | |//
    | | |
    | | |
    | | |
    | | |
    | | |
    ( ( |
    | | |
    | | |
    | | |
    | | |
    | | |
 ___[_[_]
/ /      \ 
\_\______/ 
 Fir hum dono bhik mangenge

**Comment by SirZeck** [comment]
> I will take this as a challenge , i will heck you . I will burn your billion dollar company(The one you will make by referencing your password generator as unbreakable ) to the ground.
> 
> _\/|(/_
> 
> > _   _ /
> > (_)-(_)
> > ) o (
> > \ = /
> > |W|
> > | |
> > **| |**
> > / \ u / 
> >  |   `-'   |
> > |**|   |**|
> > |||ADM|||
> > |||   |||
> > |||   |||
> > |||   |||
> > |||   |||
> > |||   |||
> > /  |_**/  
> >  ||(|   )||
> > \| | |//
> > | | |
> > | | |
> > | | |
> > | | |
> > | | |
> > ( ( |
> > | | |
> > | | |
> > | | |
> > | | |
> > | | |
> > ___[_[_]
> > / /      \
> > ___**__/
> > Fir hum dono bhik mangenge

sounds like a plan, adventure then slow living.. lesgooo
